ompi/attribute/attribute.h

```diff
@@ -155,17 +136,17 @@ typedef union ompi_attribute_fortran_ptr_t ompi_attribute_fortran_ptr_t;
 
 struct ompi_attribute_keyval_t {
     opal_object_t super;
-    ompi_attribute_type_t attr_type; /**< One of COMM/WIN/DTYPE. This
-                                       will be used to cast the
-                                       copy/delete attribute functions
-                                       properly and error checking */
-    int attr_flag; /**< flag field: contains "OMPI_KEYVAL_PREDEFINED",
-                      "OMPI_KEYVAL_F77"  */
+    ompi_attribute_type_t attr_type;            /**< One of COMM/WIN/DTYPE. This
```

Comment:
why the extra spaces?  makes no sense.

Reply:
Same as below. Aligning comments. We already do this in most of the code bar.